src/utils/modelProviders/LLMProvider.ts

```diff
@@ -174,16 +182,33 @@ export const selectBestModel = (
   //     return conversationModel
   //   }
   // }
+  const defaultModelId = localStorage.getItem('defaultModel')
+  // console.log('defaultModelId from localstorage: ', defaultModelId)
+  if (defaultModelId && allModels.find((m) => m.id === defaultModelId)) {
+    const defaultModel = allModels
+      .filter((model) => model.enabled)
```

Comment:
you have to filter for both `provider.enabled` and `model.enabled` for it to work.

Reply:
Filtered at https://github.com/KastanDay/ai-ta-frontend/blob/aa88d89ad14183cef9765dff6005666e93b28834/src/utils/modelProviders/LLMProvider.ts#L167